config/sidekiq.yml

```
@@ -14,4 +14,4 @@ production:
   - bulk
 :limits:
   bulk: 4
-  default: 84
+  default: 144
```

is it worth adding an inline comment explaining where this came from?

```suggestion
  default: 144 # = 8 workers multiplied by production `concurrency` rate 
```

---

It depends how many replicas we have as configured in the helm charts, a number I expect to drop once thi sis over, so probably not. I will add some docs after this is over to explain some of the configs though